src/App.jsx

```diff
@@ -1,16 +1,36 @@
-import React, { useState } from "react";
-import "./App.css";
-import DogList from "./components/DogList";
+import { useState, useEffect } from 'react';
+import './App.css';
+import axios from 'axios';
+import DogList from './components/DogList';
 
-const DOGS = [
-  { id: 1, name: "flasky", age: "1", breed: "golden doodle", chip: "5388" },
```

Comment:
I _just_ noticed that age is represented with a string in the data and then also in the results from the API. That seems odd, but I don't think it's worth updating the backend so a string it shall be. But I'd expect us during the live code to be a bit more cognizant of the property types we're using.